tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -53,3 +53,31 @@ def _evaluate(self, result: Mapping[str, Any]) -> Any:
         if "result" not in result:
             raise ScriptEvaluateResultException(result)
         return result["result"]
+
+    @command
+    def call_function(self,
```

Comment:
nitpick: looking at other modules in the same folder, commands are alphabetically sorted, so let's move `call_function` + `_call_function` before `evaluate.`

Reply:
done

Reply:
done